### DISCUSSION TITLE
How to enable dark mode?

### DISCUSSION THREAD

**Discussion by avin-kavish** [plouc/nivo#2387] [Q&A] [answered]
I'm wondering if nivo charts have a dark mode I can enable and all the labels will switch to legible color on dark backgrounds?

**Comment by plouc** [comment, marked as answer]
Nivo doesn't provide a dark mode feature out ouf the box, but that's something you could achieve by passing a custom theme depending on the mode.

**Reply by myudak** [reply]
but for sankey chart for example how do i custom for links color,because when in dark mode sankey links color just goes to black?? with any theme possible

EDIT: nvm i can just edit  linkBlendMode to "normal" lmfaoo
